README.md

```diff
@@ -152,5 +152,6 @@ $client->profiles->getProfileMetricTimeline( 'ProfileId', 'MetricId' );
 ```
 
 ## Rate Limiting
-  If a rate limit happens it will throw a Klaviyo/Exception/KlaviyoRateLimitException
-  This will contain a detail key with a string value mentioning the time to back off in seconds
+  If a rate limit happens it will throw a Klaviyo/Exception/KlaviyoRateLimitException.
+  After you catch this exception you can use getMessage() and it will return an array:
+  `{"detail":"Request was throttled. Expected available in 26.0 seconds.","retryAfter":26}`
```

Comment:
```suggestion
  After you catch this exception you can use getMessage() and it will return a JSON encoded array:
  `{"detail":"Request was throttled. Expected available in 26.0 seconds.","retryAfter":26}`
```